site/src/components/SSHButton/SSHButton.tsx

```diff
@@ -0,0 +1,102 @@
+import Button from "@material-ui/core/Button"
+import Popover from "@material-ui/core/Popover"
+import { makeStyles } from "@material-ui/core/styles"
+import CloudIcon from "@material-ui/icons/CloudOutlined"
+import { useRef, useState } from "react"
+import { CodeExample } from "../CodeExample/CodeExample"
+import { Stack } from "../Stack/Stack"
+import { HelpTooltipLink, HelpTooltipLinksGroup, HelpTooltipText } from "../Tooltips/HelpTooltip"
+
+export interface SSHButtonProps {
+  workspaceName: string
+  agentName: string
+  defaultIsOpen?: boolean
+}
+
+export const SSHButton: React.FC<SSHButtonProps> = ({
+  workspaceName,
+  agentName,
+  defaultIsOpen = false,
+}) => {
+  const anchorRef = useRef<HTMLButtonElement>(null)
+  const [isOpen, setIsOpen] = useState(defaultIsOpen)
+  const id = isOpen ? "schedule-popover" : undefined
+  const styles = useStyles()
+
+  const onClose = () => {
+    setIsOpen(false)
+  }
+
+  return (
+    <>
+      <Button
+        startIcon={<CloudIcon />}
+        size="small"
+        ref={anchorRef}
+        onClick={() => {
+          setIsOpen(true)
+        }}
+      >
+        SSH
+      </Button>
+      <Popover
+        classes={{ paper: styles.popoverPaper }}
+        id={id}
+        open={isOpen}
+        anchorEl={anchorRef.current}
+        onClose={onClose}
+        anchorOrigin={{
+          vertical: "bottom",
+          horizontal: "left",
+        }}
+        transformOrigin={{
+          vertical: "top",
+          horizontal: "left",
+        }}
+      >
+        <HelpTooltipText>
+          To connect with SSH you have to run the following commands:
+        </HelpTooltipText>
+
+        <Stack spacing={0.5} className={styles.codeExamples}>
+          <div>
+            <HelpTooltipText>
+              <strong className={styles.codeExampleLabel}>Configure ssh</strong>
+            </HelpTooltipText>
+            <CodeExample code="coder config-ssh" />
+          </div>
+
+          <div>
+            <HelpTooltipText>
+              <strong className={styles.codeExampleLabel}>Connect to the agent</strong>
+            </HelpTooltipText>
+            <CodeExample code={`ssh coder.${workspaceName}.${agentName}`} />
+          </div>
+        </Stack>
+
+        <HelpTooltipLinksGroup>
```

Comment:
What do you think about having some text before these links introducing them as docs? Like "For more information:" or something like that.

Reply:
Hm... I would add if the links are not explicit enough.